src/onnx/parse_resize.cpp

```diff
@@ -126,8 +123,8 @@ calc_neighbor_points(const std::vector<std::vector<std::vector<std::size_t>>>& v
                        });
     }
 
-    const auto& vv_hi = vv_ind.at(1);
-    for(std::size_t start = 0; start < vec_dims.size(); start += vv_lo.size())
+    const auto& vv_hi = vvv_ind[i_dim][1];
+    for(std::size_t start = 0; start < vec_dims.size(); start += vv_hi.size())
```

Comment:
```suggestion
    for(std::size_t start = 0; start < vec_dims.size(); start += vv_lo.size())
```
Should it be `vv_lo` ?

Reply:
They are both of same size: @ line 330, 331.